src/components/date-picker/DatePicker.tsx

```diff
@@ -276,12 +287,33 @@ class DatePicker extends Component<
         },
       ],
     });
+
+
+    if (menuShouldBlockScroll) {
+      const scrollContainer = getScrollParent(refContainer);
+      if (scrollContainer) {
+        scrollContainer.addEventListener('touchmove',this.handleScrollParent);
+        scrollContainer.addEventListener('DOMMouseScroll',this.handleScrollParent);
+        scrollContainer.addEventListener('mousewheel',this.handleScrollParent);
```

Comment:
Declare 'touchmove', 'DOMMouseScroll', mousewheel' in some const/enum in a utility file (Like keyUtils.tsx)

Reply:
ok